styles.css

```diff
@@ -122,3 +122,21 @@
 body {
   @apply font-sans text-base text-f1-foreground antialiased;
 }
+
+@media (max-width: 992px) {
+  .widget-strip > div {
+    @apply min-w-96;
+  }
+}
```

Comment:
You should be able to achieve the same behavior by using this class: `lg:[&>div]:min-w-96`

Reply:
In any case, I would use Container Queries. We do have support for it and are using it already!

https://github.com/tailwindlabs/tailwindcss-container-queries

The benfit is that you would care about the size of the dashboard itself, not the whole page. So if we change the sidebar's width for example, things would behave as expected.

Reply:
Makes total sense! I was wrongly assuming that this layout is always gonna take the same space without considering the sidebar, I'm gonna use container queries!